### PR TITLE
UI component Pull request

### PR DESCRIPTION
**Pull request by HERMON-1995**
#### => In this project, I continued with the development of the Math Magicians app. I developed a React component that will hold the core functionality: a calculator.

     - In Calculator.js, I created a React component that builds the calculator's design.
     - Implemented the design only (JSX and CSS).
     - Developed just the UI of the calculator.

### CODE REVIEW

File: src/component/calculator.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <button type="button">1</button>
+        <button type="button">2</button>
+        <button type="button">3</button>
+        <button type="button" className="operator">+</button>
+
+        <button type="button" className="zero">0</button>
+        <button type="button">.</button>
+        <button type="button" className="operator">=</button>
+
+      </div>
```
Comment (IngDavidTM) [inline src/component/calculator.js]: - [ ] You did a great job developing `Calculator.js` :+1:, but as per the project requirements, you need to create at least one child component for `Calculator.js`.

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @HERMON-1995  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [ ] You have done an excellent job in incorporating the components into your project :+1:. However, it is mandatory for the project that the folder be named `components` instead of `component`.

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @HERMON-1995 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/component/calculator.js (hunk 1 above)
Comment (HERMON-1995) [reply src/component/calculator.js]: Thanks for your feedback! I understand that I need to create a child component for Calculator.js as per project requirements, and I'll get on it right away. 😊

**Comment by HERMON-1995** [comment]
Thanks for approving my project! 😁

**Pull request merged**